html_css_dom/axios+async-await/README.md

```diff
@@ -99,28 +115,24 @@ async function returnOne() {
 }
 
 ```
-This function will returned a resolved Promise with the result of `1`.
+This function will return a resolved Promise with the result of `1`.
 
-Copy the following code into your test.js and try it out:
+Copy the following code into your JS file and try it out:
 
 ```js
-document.addEventListener('DOMContentLoaded', () => {
-
-  const returnOne = async () => { // this is how use async with ES6 syntax
+  const returnOne = async () => { // this is how to use async with ES6 syntax
     return 1;
   }
 
-  returnOne().then(alert);
-});
+returnOne().then(alert);
 
 
 ```
-
 ### Await
 There is also a keyword called `await`. `await ` only works inside `async` functions and makes JavaScript wait until that promise
 settles and returns its result. Let's see this in action.
 
-First, let's see how things work without `await`. Create a function called fireRequest inside your test.js file.
+First, let's see how things work without `await`. Create a function called fireRequest inside your JS file.
 Use axios to make a get request to https://jsonplaceholder.typicode.com/posts. Save the request to a variable called response and
 have your function console.log the response on the next line.
```

Comment:
Before fellows run the code, can we have them predict what `console.log(response)` will print to the console? This is a good potential zoom poll topic and a check of how well the class understands async JS principles so far. I am guessing that many / most will predict that it logs the API response (not a pending Promise)

Reply:
Yeah they definitely should always be predicting what every line of code they write will evaluate too. I can specify that in the readme here.